website/docs/sources/ldap/index.md

```diff
@@ -2,41 +2,74 @@
+
+    -   **Enable StartTLS**: Enables StartTLS functionality. To use LDAPS instead, use port `636`.
+    -   **Use Server URI for SNI verification**: this setting is required for servers using TLS 1.3+
+
+-   **TLS Verification Certificate**: Specify a keypair to validate the remote certificate.
+
+-   **TLS Client authentication**: Client certificate keypair to authenticate against the LDAP Server's Certificate.
 
 -   **Bind CN**: CN of the bind user. This can also be a UPN in the format of `user@domain.tld`.
+
 -   **Bind password**: Password used during the bind process.
--   **Enable StartTLS**: Enables StartTLS functionality. To use LDAPS instead, use port `636`.
--   **Base DN**: Base DN used for all LDAP queries.
+
+-   **Base DN**: Base DN (distinguished name) used for all LDAP queries.
+
+#### LDAP Attribute mapping
+
+-   **User Property mappings** and **Group Property Mappings**: Define which LDAP properties map to which authentik properties. The default set of property mappings is generated for Active Directory. See also [LDAP Property Mappings](../../../docs/property-mappings/#ldap-property-mapping).
+
+#### Additional Settings
+
+-   **Group**: Parent group for all the groups imported from LDAP.
+
+-   **User path**: Path template for all new users created.
+
 -   **Addition User DN**: Prepended to the base DN for user queries.
+
 -   **Addition Group DN**: Prepended to the base DN for group queries.
+
 -   **User object filter**: Consider objects matching this filter to be users.
+
 -   **Group object filter**: Consider objects matching this filter to be groups.
--   **User group membership field**: This field contains the user's group memberships.
+
+-   **Group membership field**: This field contains the user's group memberships.
+
 -   **Object uniqueness field**: This field contains a unique identifier.
--   **Sync groups**: Enable/disable group synchronization. Groups are synced in the background every 5 minutes.
--   **Sync parent group**: Optionally set this group as the parent group for all synced groups. An example use case of this would be to import Active Directory groups under a root `imported-from-ad` group.
--   **Property mappings** and **Group Property Mappings**: Define which LDAP properties map to which authentik properties. The default set of property mappings is generated for Active Directory. See also [LDAP Property Mappings](../../../docs/property-mappings/#ldap-property-mapping)
 
 ## Property mappings
 
 LDAP property mappings can be used to convert the raw LDAP response into an authentik user/group.
 
-By default, authentik ships with some pre-configured mappings for the most common LDAP setups.
+By default, authentik ships with some [pre-configured mappings](../../property-mappings/index.md#ldap-property-mapping) for the most common LDAP setups.
```

Comment:
Impression: "some" weakens the impact.  Removing it makes the sentence clear:

"authentik ships with pre-configured mappings for the most common LDAP setups."

(... "These mappings can be found on the LDAP Source Configuration page.", maybe?)